applications/views_test.py

```diff
@@ -194,39 +202,84 @@ def test_review_submission_list(admin_drf_client, submission_factory):
             for submission in sorted(submissions, key=lambda s: s.id)
         ],
         "facets": {
-            "bootcamps": [
+            "bootcamp_runs": [
                 {
-                    "id": bootcamp.id,
-                    "title": bootcamp.title,
-                    "count": len(ALL_REVIEW_STATUSES),
+                    "id": bootcamp_run.id,
+                    "title": bootcamp_run.title,
+                    "start_date": serializer_date_format(bootcamp_run.start_date),
+                    "end_date": serializer_date_format(bootcamp_run.end_date),
+                    "count": 4,
                 }
-                for bootcamp in sorted(bootcamps, key=lambda b: b.id)
+                for bootcamp_run in sorted(bootcamp_runs, key=lambda b: b.id)
             ],
             "review_statuses": [
-                {"review_status": review_status, "count": len(bootcamps)}
+                {"review_status": review_status, "count": len(bootcamp_runs)}
                 for review_status in sorted(ALL_REVIEW_STATUSES)
             ],
         },
     }
 
 
-def test_review_submission_list_query_bootcamp_id(admin_drf_client):
+def test_review_submission_list_expired(admin_drf_client):
+    """
+    The review submission list view should not return submissions if run ended
+    """
+    bootcamp_runs = BootcampRunFactory.create_batch(
+        3, end_date=now_in_utc() - timedelta(weeks=1)
+    )
+    submissions = []
+    for bootcamp_run in bootcamp_runs:
+        application_step = BootcampRunApplicationStepFactory(
+            bootcamp_run=bootcamp_run, application_step__bootcamp=bootcamp_run.bootcamp
+        )
+        for review_status in ALL_REVIEW_STATUSES:
+            submissions.append(
+                ApplicationStepSubmissionFactory.create(
+                    review_status=review_status,
+                    bootcamp_application__bootcamp_run=bootcamp_run,
+                    run_application_step=application_step,
+                    content_object=VideoInterviewSubmissionFactory.create(),
+                    is_review_ready=True,
+                )
+            )
```

Comment:
Can you move this data setup to a fixture and wrap the individual elements into a `SimpleNamespace` or `namedtuple`?